.github/workflows/tests.yml

```diff
@@ -262,7 +262,7 @@ jobs:
           script: |
             const workflow_url = `https://github.com/${process.env.GITHUB_REPOSITORY}/actions/runs/${process.env.GITHUB_RUN_ID}`
             const issue_body = `[Workflow Run URL](${workflow_url})`
-            github.issues.create({
+            github.rest.issues.create({
                 owner: context.repo.owner,
                 repo: context.repo.repo,
                 body: issue_body,
```

Comment:
Don't need to do anything here, but just wanted to note that the original version of this code has been moved into it's own GitHub action https://github.com/xarray-contrib/issue-from-pytest-log which is what we're now using in `dask/dask`

https://github.com/dask/dask/blob/98331028f8a51d404f8c9856d49ba726322c4e03/.github/workflows/upstream.yml#L76-L86

Reply:
To be clear, this is mostly just an FYI